templates/404.html

```diff
@@ -0,0 +1,10 @@
+<!DOCTYPE html>
```

Comment:
Nie widzę żebyś używał tego templatu. Pewnie go w serwerze gdzieś włączasz co? A w Django można to fajnie obsłuzyć. W pliku z urlsami umieszczasz takie zmienne:
```
handler400 = errors.Custom400Handler.as_view()
handler403 = errors.Custom403Handler.as_view()
handler404 = errors.Custom404Handler.as_view()
handler500 = errors.Custom500Handler.as_view()
```

a przykładowy widok wygląda tak
```
class Custom400Handler(TemplateView):
    template_name = 'errors/400.html'
```

Reply:
Nie wdrażałem tego jeszcze w ogóle. Template stworzyłem testowo i po prostu został... Dzięki za wskazówkę.